Wabbajack/UI/ReactiveMetro.cs

```diff
@@ -4,23 +4,43 @@
 
 namespace Wabbajack.UI
 {
+    /// <summary>
+    /// A <see cref="MetroWindow"/> that is reactive.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// This class is a <see cref="MetroWindow"/> that is also reactive. That is, it implements <see cref="IViewFor{TViewModel}"/>.
+    /// You can extend this class to get an implementation of <see cref="IViewFor{TViewModel}"/> rather than writing one yourself.
```

Comment:
Maybe take this one comment line out?  I don't expect anyone to inherit from MetroWindow too often.  Any other view that wants `IViewFor` probably won't be a window and should just inherit `ReactiveUserControl`, or one that only implementes IViewFor, without getting the MetroWindow logic?